portfolio/src/main/webapp/style.css

```diff
@@ -26,17 +26,17 @@ body {
 }
 
 .hand {
+  display: inline;
   font-weight: 800;
   font-size: 50px;
-  display: inline;
 }
 
 #my-pic {
```

Comment:
Use CSS Class instead of id

Reply:
for #my-pic or some particular element or for all?